src/main/java/io/codelex/arrays/exercises/Exercise_7.java

```diff
@@ -0,0 +1,90 @@
+package io.codelex.arrays.exercises;
+
+import java.util.Scanner;
+
+public class Exercise_7 {
+    public static void main(String[] args) {
+        ticTacToe();
+    }
+
+    private static void ticTacToe() {
+        Scanner scan = new Scanner(System.in);
+        char[][] gameBoardArray = new char[3][3];
+        for (int i = 0; i < 3; i++) {
+            for (int j = 0; j < 3; j++) {
+                gameBoardArray[i][j] = '-';
+            }
+        }
+        String gameBoardString = printBoard(gameBoardArray);
+        System.out.println(gameBoardString);
+
+        char xOrO = 'X';
```

Comment:
šeit jau vari sākotnēji nepiešķirt vērtību, 28 vai 30 rindiņā jau tāpat piešķier vērtibu pirms kaut ko dari ar to